core/src/poh_service.rs

```diff
@@ -199,6 +202,7 @@ impl PohService {
         }
     }
 
+    // returns target time for tick if we need to tick, or None if we don't need to tick
```

Comment:
hmm, does it return through an argument? Doesn't this just return `bool`?

Reply:
yes, comment was stale